guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<p> </p><blockquote><a href="https://developers.arcgis.com/python/samples/04_gis_analysts_data_scientists/climate_change_impacts_on_trees_in_delhi" target="_blank">Climate Change Impacts on Trees in Delhi</a> - This can be removed as this is just a case study for now and we may have sample later but not possible to ship it with 17.1 release.</blockquote><p>sample <a href="https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/how-much-green-is-Delhi-as-on-15-oct-2017.ipynb" target="_blank">https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/how-much-green-is-Delhi-as-on-15-oct-2017.ipynb</a> ,<a href=" https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/spatial_and_temporal_trends_of_service_calls.ipynb" target="_blank"> https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/spatial_and_temporal_trends_of_service_calls.ipynb</a> and <a href="https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/safe_streets_to_schools.ipynb" target="_blank">https://github.com/Esri/arcgis-python-api/blob/master/samples/04_gis_analysts_data_scientists/safe_streets_to_schools.ipynb</a> needs to be added to this list.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>made the approriate changes</p>